### DISCUSSION TITLE
UI chat window: overflow-y scroll bar almost impossible to target/drag sideways as it's too tiny

### DISCUSSION THREAD

**Discussion by vladimirsbelajevs** [danny-avila/LibreChat#2455] [Feature Requests & Suggestions]
### What happened?

The scroll bar in \<code\>\</code\> wrapper \<div\>\</div\> is too small, it's impossible to use it to scroll horizontally 

### Steps to Reproduce

1. Start new chat
2. Ask any AI to generate code snippet that is longer than avail horizontal space
3. Try to scroll that code snippet sideways

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### Screenshots

The ::-webkit-scrollbar has height: .1em which causes really small scrollbar
![image](https://github.com/danny-avila/LibreChat/assets/4044693/e658ae23-8a69-4d55-bcd8-abcb992b8e13)

If we increase it to .5em - .7em then it's more usable
![image](https://github.com/danny-avila/LibreChat/assets/4044693/5df035e1-1a87-4515-9847-2c4a75c72cac)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
PRs welcome
